core/capabilities/remote/dispatcher.go

```diff
@@ -114,7 +114,11 @@ func (d *dispatcher) SetReceiver(capabilityId string, donId uint32, rec remotety
 			case <-ctx.Done():
 				return
 			case msg := <-receiverCh:
-				rec.Receive(ctx, msg)
+				if msg != nil {
```

Comment:
This can't be nil because ValidateMessage() never returns nil... Do we need this PR at all?

Reply:
@bolekk when failing to unmarshal the message it returns `nil`.

It can be seen in the first couple of lines [here](https://github.com/smartcontractkit/chainlink/blob/69f7bd68199b91c4ef4be65b5701e4d45a250350/core/capabilities/remote/utils.go#L20)

Reply:
You are right, I'll close this PR.